docs/changelog.json

```diff
@@ -1,7 +1,157 @@
 {
   "url": "https://github.com/Shopify/hydrogen/pulls?q=is%3Apr+is%3Aclosed+%5Bci%5D+release+in%3Atitle+is%3Amerged",
-  "version": "2024.4.3",
+  "version": "1",
```

Comment:
oh we don't need to match this?

Reply:
No, I confirmed with @frandiox a while back this version is the version of the changelog itself. We change it when we change the changelog's format. Just been forgetting to update it.